clients/clients-integration-tests/src/test/java/org/apache/kafka/clients/consumer/ShareConsumerTest.java

```diff
@@ -3447,6 +3730,37 @@ private void alterShareIsolationLevel(String groupId, String newValue) {
         }
     }
 
+    private List<Integer> topicPartitionLeader(Admin adminClient, String topicName, int partition)
+        throws InterruptedException, ExecutionException {
```

Comment:
nit: The indentation on this method and the next is a bit odd because the throws and the start of the method body are equally indented. I'd just put the throws on the same line as the argument list for these methods.